### ISSUE TITLE
Breaking method  is not working with bigint columns

### ISSUE THREAD

**Issue by teoria** [datacontract/datacontract-cli#346]
When you compare 2 contracts with bigint and long column must be equal.

```

datacontract breaking  contract_v1.yaml contract_v2_glue.yml

1 breaking changes: 1 error, 0 warning
error   [field_type_updated] at contract_v2_glue.yml
        in my_table.fields.my_column.type
            changed from `bigint` to `long`
```

**Comment by roykoand** [comment]
@teoria I don't think that AWS Glue supports `long` data type. I've only seen `bigint` data type in their UI once tried to create a table:
<img width="631" alt="image" src="https://github.com/user-attachments/assets/9c070ccb-5159-4954-9806-8f5040ecf1a5">

<img width="657" alt="image" src="https://github.com/user-attachments/assets/41c8690c-3b14-4c37-8db3-cfab57a1fb69">

**Comment by teoria** [comment]
when i use the glue importer the code cast bigint to long
then when i call breaking  the cli identify the type update.
the breaking method need ignore bigint->long or the import need keep bigint type

**Comment by roykoand** [comment]
@teoria yes, I agree. Data contract should keep `bigint` data type. This line is a culprit:
https://github.com/datacontract/datacontract-cli/blob/56f4dbe4469c65c03b42fc6464258b6e6dd1e193/datacontract/imports/glue_importer.py#L268

**Comment by teoria** [comment]
or breaking ignore the bigint->long
@jochenchrist what do you think ?

**Comment by jochenchrist** [comment]
lets change the importer to keep bigint

**Comment by jochenchrist** [comment]
fixed with #351.

**Comment by jochenchrist** [comment]
Will be part of release v0.10.12
